### PR TITLE
Feature/select picture microblog

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+
+  // This couldn't be injected by constructor because it makes
+  // the container fail on startup
+  private ManageDocumentService    documentService;
+
+  private String                   personalDriveName                   = PERSONAL_DOCUMENTS_DRIVE_NAME;
+
+  private String                   repositoryName;
+
+  public ActivityImageLinkUpdateListener(PortalContainer portalContainer,
```
Comment (thomasdelhomenie) [inline integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java]: Can you add a description on this class to explain its role please ?

File: integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+    int month = yearMonth.getMonthValue();
+    String monthString = String.format("%02d", month);
+
+    String currentFolder = null;
+    DriveData selectedDriveData = null;
+
+    if (activity.getActivityStream().getType() != null && SpaceIdentityProvider.NAME.equals(activity.getActivityStream().getType().toString())) {
+      currentFolder = SPACE_DOCUMENTS_FOLDER + "/" + year + "/" + monthString;
+      String streamOwner = activity.getStreamOwner();
+      SpaceService spaceService = portalContainer.getComponentInstanceOfType(SpaceService.class);
```
Comment (thomasdelhomenie) [inline integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java]: Can be injected in constructor.

File: integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java (hunk 1 above)
Comment (boubaker) [reply integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java]: done

File: integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java (hunk 2 above)
Comment (boubaker) [reply integ-social/integ-social-ecms/src/main/java/org/exoplatform/social/ckeditor/listener/ActivityImageLinkUpdateListener.java]: done

**Pull request merged**